src/server/page_server.cpp

```diff
@@ -232,6 +239,8 @@ page_server::connection_handler::abnormal_tran_server_disconnect (css_error_code
 			(int)error_code);
 
 	  remove_prior_sender_sink ();
+
+	  m_ps.m_pts_mvcc_tracker.delete_oldest_active_mvccid (get_channel_id());
```

Comment:
How about moving this part to `page_server::disconnect_tran_server_async ()` ? 

Then we can remove the same part in `page_server::connection_handler::receive_disconnect_request` .

Reply:
The `disconnect_tran_server_async()` is a different job. To put it together, I think we should make a wrapper function like `cleanup_tran_server()` calling them. It would be better to make that if we get more thing to clean up later.

Reply:
OK. I just mentioned the above comment because both functions (`abnormal_tran_server_disconnect()`, and `receive_disconnect_request()` ) commonly call the `disconnect_tran_server_async()` function.